### DISCUSSION TITLE
Error when running pgsc_calc

### DISCUSSION THREAD

**Discussion by PatrickReck01** [PGScatalog/pgsc_calc#226] [Q&A] [answered]
I get the following error when trying to run the tool with a vcf file with a single sample in it and I get the following error:

<img width="1179" alt="Screenshot 2023-12-30 at 22 23 06" src="https://github.com/PGScatalog/pgsc_calc/assets/134544998/926720ce-7d82-4b75-bcf9-e39f23edd12d">
<img width="1156" alt="Screenshot 2023-12-30 at 22 23 23" src="https://github.com/PGScatalog/pgsc_calc/assets/134544998/9ef6eaee-ac89-45a3-a043-a7d88f4bf44c">

Any idea what this error means and how I can fix it? The sample I am using has variants from all chromosomes in it. When using the same sample, but in a file that has only variants from a few chromosomes in it, it works.

Edit: I just tested something: When running the tool with a separate file for each chromosome (still only 1 sample in each) it works, but when running the tool with the combined vcf that has every chromosome in it it still doesn't work.

**Comment by smlmbrt** [comment, marked as answer]
Hi @PatrickReck01 - the 137 error code happens when the job/pipeline is out of memory (RAM). See https://pgsc-calc.readthedocs.io/en/latest/how-to/bigjob.html for information about how to run with more memory. If you're running on a laptop it's also worth checking that docker is allowed to make containers of the size you're requesting (https://docs.docker.com/config/containers/resource_constraints/; and on a Mac: https://docs.docker.com/desktop/settings/mac/).
